.travis.yml

```diff
@@ -80,7 +80,7 @@ before_install:
     _travis_terminate_unix
     _travis_terminate_windows
   # install/run nengo-bones
-  - pip install git+https://github.com/nengo/nengo-bones.git@$(git rev-parse --abbrev-ref HEAD)
+  - pip install git+https://github.com/nengo/nengo-bones.git@$(git symbolic-ref --short HEAD)
```

Comment:
Just a note for merge, this should be fixing up "Make nengo-bones version configurable", not "Add more documentation", right?